js/src/components/lineage/NodeSqlView.tsx

```diff
@@ -95,7 +96,15 @@ export const NodeSqlView = ({ node }: NodeSqlViewProps) => {
         <ModalOverlay />
         <ModalContent overflowY="auto" height="75%">
           <ModalHeader>
-            {isSingleEnvOnboarding ? "Model Raw Code" : "Model Raw Code Diff"}
+            {isSingleEnvOnboarding ? (
+              <h1>
```

Comment:
Hi Kent,
Does the `Model code` need to be wrapped by `<h1>`?
(There's no `<h1>` for Model code diff)

If yes, would it be more consistent to use the Chakra component?
```
<Heading as="h1">
```

Reply:
Thanks for suggesting, I will change to use `<heading as="h1" size="md">` for both singe env and normal mode